### PR TITLE
todo: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.nl/common/todo.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# todo
+
+> Een eenvoudige, op standaarden gebaseerde, opdrachtregel todo manger.
```
Comment (leonvsc) [inline pages.nl/common/todo.md:3]:
```suggestion
> Een eenvoudige, op standaarden gebaseerde, opdrachtregel todo manager.
```

File: pages.nl/common/todo.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`todo list --startable`
+
+- Voeg een nieuwe taak toe aan de werklijst:
+
+`todo new {{ding_om_te_doen}} --list {{werk}}`
+
+- Voeg een locatie toe aan een taak met een gegeven ID:
+
+`todo edit --location {{locatie_name}} {{taak_id}}`
```
Comment (leonvsc) [inline pages.nl/common/todo.md:16]:
```suggestion
`todo edit --location {{locatie_naam}} {{taak_id}}`
```

File: pages.nl/common/todo.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Toon details over een task:
```
Comment (leonvsc) [inline pages.nl/common/todo.md:18]:
```suggestion
- Toon details over een taak:
```

**Review by leonvsc — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.

LGTM, after @leonvsc's suggestions.

**Pull request merged**